### DISCUSSION TITLE
Notification for install

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [Droid-ify/client#76] [General]
Even though app got installed there is still notification tap to install the file?

How about autoremoving or something

**Comment by Iamlooker** [comment]
This has been fixed

**Reply by HikaruIchijyo** [reply]
Got it

**Comment by HikaruIchijyo** [comment]
now we have installed notification🤔

**Reply by Iamlooker** [reply]
Yes, it appears unless you are using Shizuku/Root installer.

**Reply by HikaruIchijyo** [reply]
> Yes, it appears unless you are using Shizuku/Root installer.

ahaaa got it

**Comment by Iamlooker** [comment]
I think we can close this 😅

**Reply by HikaruIchijyo** [reply]
ofc
